tests/ome_tiff.rs

```diff
@@ -0,0 +1,21 @@
+/// Integration tests on OME-TIFF files.
+use async_tiff::tiff::tags::PhotometricInterpretation;
+
+mod util;
+
+#[tokio::test]
+async fn test_ome_tiff_single_channel() {
+    let tiff =
+        util::open_remote_tiff("https://downloads.openmicroscopy.org/images/OME-TIFF/2016-06/bioformats-artificial/single-channel.ome.tif").await;
```

Comment:
yeah I think it would be better to either check in this file into this repo, or perhaps create an `async-tiff-fixtures` separate repo that we can check data into.

Assuming this file is large, it would be great if we could extract like a one-pixel or very small subset of this file, while maintaining all header metadata the same.

_Or_, we could find the size of the last IFD and just clip the existing buffer to that region. So all IFD parsing would succeed but any image loading would fail as the image data is past the byte region checked in to the test fixture.